composites/SnippetPreview/tests/IconTest.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import renderer from "react-test-renderer";
+
+import Edit from "../../../style-guide/svg/edit.svg";
```

Comment:
Technically it is a component since we use `svg-react-loader`, but for clarity i agree lowercase is better.